agent/devmock/avl/tree.go

```diff
@@ -0,0 +1,82 @@
+package avl
```

Comment:
I believe the Go agent should not need to be aware of the avl.Tree structure and should instead focus on passing slices of types.

Reply:
It doesn't; this was for local development so I could get code completion in my editor